docs/reference/contrib/sitemaps.rst

```diff
@@ -84,7 +84,10 @@ Each dictionary can contain the following:
  - **changefreq**
  - **priority**
 
-You can add more but you will need to override the ``wagtailsitemaps/sitemap.xml`` template in order for them to be displayed in the sitemap.
+You can add more but you will need to override the ``wagtailsitemaps/sitemap.xml`` template in order for them to be displayed in the sitemap, extending the ``Sitemap`` generator is also an option.
+
+.. note::
+    As of Wagtail 1.10 it's possible to extend te existing ``Sitemap`` generator class. this can be done by setting ``WAGTAILSITEMAPS_GENERATOR`` to your own implementation of the ``Sitemap`` object. This can be used to implement multi language sitemaps for example.
```

Comment:
te existing -> the existing
this can be done -> This can be done
sitemaps for example -> sitemaps, for example